master/internal/resourcemanagers/resource_pool.go

```diff
@@ -377,15 +378,19 @@ func (rp *ResourcePool) receiveAgentMsg(ctx *actor.Context) error {
 
 func (rp *ResourcePool) moveJob(msg job.MoveJob) error {
 	// find out what is the job before or after the anchor
-	queueInfo := rp.scheduler.JobQInfo(rp)
+	// queueInfo := rp.scheduler.JobQInfo(rp)
 
-	newPos, err := computeNewJobPos(msg, queueInfo, rp.queuePositions)
+	fmt.Println(rp.config.PoolName, "moveJob: ", msg)
+	rp.queuePositions[job.TailAnchor] = initalizeQueuePosition(time.Now()) // x
+	newPos, rebalance, err := computeNewJobPos(msg, rp.queuePositions)
```

Comment:
> Is there a way we could check for precision?

I've kinda left this as a future problem since the pr was already getting big. we should be able to come up with the correct answer.